app/components/file_attachment_list/file_attachment_document.js

```diff
@@ -357,21 +367,14 @@ export default class FileAttachmentDocument extends PureComponent {
                 </CircularProgress>
             );
         } else {
-            fileAttachmentComponent = (
-                <FileAttachmentIcon
-                    file={file.data}
-                    theme={theme}
-                    iconHeight={iconHeight}
-                    iconWidth={iconWidth}
-                    wrapperHeight={wrapperHeight}
-                    wrapperWidth={wrapperWidth}
-                />
-            );
+            fileAttachmentComponent = this.renderFileAttachmentIcon();
         }
 
         return (
             <TouchableOpacity onPress={this.handlePreviewPress}>
-                {fileAttachmentComponent}
+                <View style={{backgroundColor: '#fff'}}>
```

Comment:
Can this go in the stylesheet?

Reply:
I think inline styles are better for this as it is not generic to move this to the stylesheet and does not have any theme constraints. 

Should we always move styles to the stylesheet?

Reply:
Unless we're dynamically styling the component, they should always go into the stylesheet. Generally, using an entry in a stylesheet is slightly faster due to how RN connected the JavaScript to the native code